pkg/apis/fiaas.schibsted.io/v1/doc.go

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2016 The Kubernetes Authors.
```

Comment:
This copyright does not look right.

Reply:
I see the same problem in several files. We should probably check that we have the right copyright everywhere.